virtme_ng/run.py

```diff
@@ -272,6 +272,11 @@ def make_parser():
     parser.add_argument(
         "--force-9p", action="store_true", help="Use legacy 9p filesystem as rootfs"
     )
+    parser.add_argument(
+        "--force-tcp",
```

Comment:
same here: part of the `Remote Console` group and `--force-ssh-tcp`?

Reply:
I've moved `--force-tcp` to the `Remote Console` group, but have not renamed it.

@arighi Opinions?